sdcm/utils/issues.py

```diff
@@ -88,6 +88,9 @@ def get_issue_details(self, issue):
                                severity=Severity.WARNING,
                                trace=sys._getframe().f_back).publish()  # pylint: disable=protected-access
             return None
+        except RateLimitExceededException as exc:
+            logging.debug('RateLimitExceededException raise: %s', str(exc))
```

Comment:
Why `debug`? It should be visible in jenkins for fast debugging.
Please, make it be `warning`.

Reply:
no, I don't see the need for it.

we shouldn't spam the output with things are not really important, this isn't really important